src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.ts

```diff
@@ -128,6 +136,23 @@ export class InstructorCourseStudentEditPageComponent implements OnInit, OnDestr
             .subscribe(() => this.isEmailFieldChanged = true);
   }
 
+  /**
+   * Displays message to user stating that the field is empty.
+   */
+  displayEmptyFieldMessage(fieldName: string, maxLength: number): string {
+    return `The field '${fieldName}' is empty.` +
+        ` The value of a/an ${fieldName} should be no longer than ${maxLength} characters.` +
+        ' It should not be empty.';
+  }
+
+  /**
+   * Displays message to user stating that the field exceeds the max length.
+   */
+  displayExceedMaxLengthMessage(fieldName: string, maxLength: number): string {
+    return `The input is not acceptable to TEAMMATES as a/an '${fieldName}'` +
```

Comment:
Perhaps `${fieldName} should not exceed ${maxLength} characters.` would be better?

Reply:
Same comment as https://github.com/TEAMMATES/teammates/pull/9633#discussion_r272031420